streams/src/test/java/org/apache/kafka/streams/integration/IQv2StoreIntegrationTest.java

```diff
@@ -1586,12 +1593,15 @@ public <V> void shouldHandleKeyQuery(
     public <V> void shouldHandleRangeQuery(
         final Optional<Integer> lower,
         final Optional<Integer> upper,
+        final boolean isKeyAscending,
         final Function<V, Integer> valueExtactor,
-        final Set<Integer> expectedValue) {
+        final List<Integer> expectedValue) {
```

Comment:
Nit: `expectedValue[s]`

Reply:
ok, I will update it.

Reply:
It seems that exiting code using expectedValue and actualValue for other methods, I will update them all.